packages/migrate/Migration.js

```diff
@@ -185,6 +185,14 @@ class Migration {
    * @param  {Object}   options  config and command-line
    */
   async run(options) {
+    // check --migrate-none
+    if (config.migrateNone || config['migrate-none']) {
```

Comment:
Style note: We use 2-space indenting here, not 4. :)  We used to have `prettier` handle this sort of thing for you, but, uh, husky hasn't been working in quite a while (#3530), so we're stuck doing it ourselves for now. :-/  I believe @cds-amal was working on fixing this, but I'm not sure what the current status of that is.